packages/core/src/components/Button/Button.scss

```diff
@@ -174,38 +150,17 @@
   }
 }
 
-.ds-c-button--disabled {
```

Comment:
Because we added `.ds-c-button--disabled` to all the `&:disabled`, we can remove this separate class definition